nixos/modules/services/system/earlyoom.nix

```diff
@@ -67,9 +67,19 @@ in
       notificationsCommand = mkOption {
         type = types.nullOr types.str;
         default = null;
-        example = "sudo -u example_user DISPLAY=:0 DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus notify-send";
         description = ''
-          Command used to send notifications.
+          This option is deprecated and ignored by earlyoom since 1.6.
+          Use <option>services.earlyoom.enableNotifications</option> instead.
+        '';
+      };
+
+      enableNotifications = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Send notifications about killed processes via the system d-bus.
+          To actually see the notifications in your GUI session, you need to have
+          <literal>systembus-notify</literal> running as your user.
```

Comment:
Maybe we should package this? Opened https://github.com/rfjakob/systembus-notify/issues/3 for a user service

Reply:
See #88441. But is there any way to setup a systemd user service in system configuration?

Reply:
It sounds like the autostart desktop file is enough

Reply:
(but yes there is, through systemd.user.services I think)

Reply:
Should I make `environment.systemPackages = [ pkgs.systembus-notify ];` when notification is enabled? But this requires #88441 to be merged first.

Reply:
Yeah - let’s do that and merge systembus-notify first